python/ray/data/tests/test_delta.py

```diff
@@ -0,0 +1,77 @@
+import os
+import sys
+import tempfile
+
+import pandas as pd
+import pytest
+from pyspark.sql import SparkSession
+
+import ray.data
+from ray.data import read_delta
+
+
+def get_or_create_spark_session():
+    return (
+        SparkSession.builder.config(
+            "spark.jars.packages", "io.delta:delta-core_2.12:2.2.0"
+        )
+        .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension")
+        .config(
+            "spark.sql.catalog.spark_catalog",
+            "org.apache.spark.sql.delta.catalog.DeltaCatalog",
+        )
+        .getOrCreate()
+    )
+
+
+@pytest.fixture(scope="module")
+def spark():
+    with (get_or_create_spark_session()) as spark:
+        yield spark
+
+
+def test_read_delta_basic(spark):
```

Comment:
We should test the case:
 1. create a delta dataset and save it.
 2. update some rows in the dataset
 3. delete some rows in the dataset
 4. Using ray delta loader to load the dataset and see whether it gets latest valid rows data.
 
To do this, you can uses `sparkSession.sql` API to create delta table like `CREATE TABLE table_name USING DELTA LOCATION'/path/to/data'` , then inserting rows, updating rows, deleting rows. then using ray delta loader to load data from the table data path